problem_builder/tests/integration/test_step_builder.py

```diff
@@ -624,9 +624,7 @@ def test_review_tips(self):
 
     @data(True, False)
     def test_conditional_messages(self, include_messages):
-        """
-        Test that conditional messages in the review step are visible or not, as appropriate.
-        """
+        # Test that conditional messages in the review step are visible or not, as appropriate.
         max_attempts = 3
         extended_feedback = False
         params = {
```

Comment:
Not really, we may check whether HTML blocks are detached. 

However I'd like to have some tests for the Video XBlock functionality (but these are definitely out of scope for now).